src/Skeletor/Manager/ProviderManager.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace Skeletor\Manager;
+
+use Skeletor\Frameworks\Framework;
+use Skeletor\Packages\Package;
+
+class ProviderManager extends Manager
+{
+    const PROVIDER_METHOD = 'getProviderClass';
+    const FACADE_METHOD = 'getFacadeClass';
+
+    /**
+     * @param Package $package
+     * @param Framework $activeFramework
+     */
+    public function configure(Package $package, Framework $activeFramework)
+    {
+        if ($activeFramework->getName() !== 'Laravel') {
```

Comment:
Instead of hardcode by name, Framework should have a method like configurable() that returns a boolean. In the Laravel one it would be true, all the others false, this way we would have more flexibility in our code